scripts/flash/show-initial-reporter.js

```diff
@@ -5,10 +5,20 @@
 var chalk = require("chalk");
 var displayTime = require("./display-time");
 
+function help() {
+  console.log(chalk.red("Show initial reporter account for market"));
+  console.log(chalk.red("Displays all 0s if market hasn't been reported on"));
+}
+
 /**
  * Move time to Market end time and do initial report
  */
-function showInitialReporterInternal(augur, marketId, auth, callback) {
+function showInitialReporter(augur, args, auth, callback) {
```

Comment:
`auth` param doesn't get used here.  I didn't check every function in this PR for unused params, but if this is something that should be corrected, I recommend double-checking the other functions in this PR as well.

Reply:
Every command gets passed these parameters/options. It's like an interface if we were using typescript.